ebos/eclalugridvanguard.hh

```diff
@@ -170,17 +170,17 @@ public:
 
        if (grid().size(0))
        {
-        globalTrans_.reset(new TransmissibilityType(this->eclState(),
-                                                    this->gridView(),
-                                                    this->cartesianIndexMapper(),
-                                                    this->grid(),
-                                                    this->cellCentroids(),
-                                                    getPropValue<TypeTag,
-                                                    Properties::EnableEnergy>(),
-                                                    getPropValue<TypeTag,
-                                                    Properties::EnableDiffusion>()));
-        // Re-ordering  for ALUGrid
-        globalTrans_->update(false, [&](unsigned int i) { return gridEquilIdxToGridIdx(i);});
+           globalTrans_.reset(new TransmissibilityType(this->eclState(),
```

Comment:
If you're changing this anyway, how about switching to `make_unique<TransmissibilityType>()` instead?

Reply:
sure.